src/main/common/interestUtils.ts

```diff
@@ -13,13 +13,19 @@ import { InterestType as ClaimInterestType } from 'claims/models/interestType'
 import { YesNoOption } from 'models/yesNoOption'
 import { InterestTypeOption } from 'claim/form/models/interestType'
 
+function getInterestEndDate (issuedOn: moment.Moment, interestFromDate: moment.Moment): moment.Moment {
+  const currentDate = MomentFactory.currentDate()
+  const endDate = issuedOn > currentDate ? issuedOn : currentDate
+  return moment.max(interestFromDate, endDate)
+}
+
 export async function getInterestDetails (claim: Claim): Promise<InterestData> {
   if (claim.claimData.interest.type === ClaimInterestType.NO_INTEREST || claim.claimData.interest.type === undefined) {
     return undefined
   }
 
   const interestFromDate: moment.Moment = getInterestDateOrIssueDate(claim)
-  const interestToDate: moment.Moment = moment.max(interestFromDate, MomentFactory.currentDate())
+  const interestToDate: moment.Moment = getInterestEndDate(claim.issuedOn, interestFromDate)
```

Comment:
`getInterestEndDate` should be removed and we should just use `claim.issuedOn` here.

Reply:
@rlewan we don't stop calculating interest in the app though, only PDFs stop calculating at issue date.
so it should use either today, or if issue date is in the future use issue date

Reply:
shouldn't for daily interestRate, date should be increasing everyday
by putting interesetEndDate to issuedOn we are fixing it one date